design/search.go

```diff
@@ -34,6 +34,7 @@ var _ = a.Resource("search", func() {
 				3) "simple keywords separated by space" :- Search in Work Items based on these keywords.`)
 			a.Param("page[offset]", d.String, "Paging start position") // #428
 			a.Param("page[limit]", d.Integer, "Paging size")
+			a.Param("spaceID", d.String, "search within given Space ID if provided")
```

Comment:
roger that https://github.com/almighty/almighty-core/pull/1184/commits/a94d60674b495855a5f0c50d334c0890fa95f4e3